docs/maui/behaviors/touch-behavior.md

```diff
@@ -269,3 +269,94 @@ You can find an example of this behavior in action in the [.NET MAUI Community T
 ## API
 
 You can find the source code for `TouchBehavior` over on the [.NET MAUI Community Toolkit GitHub repository](https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui/Behaviors/TouchBehavior.shared.cs).
+
+
+## Migrating From Xamarin Community Toolkit
```

Comment:
Not sure if it's relevant but should you include a full example?

This is how your Effect looked like in Xamarin, this is the same Behavior in .NET MAUI so you can really easily spot the differences? Or are they that small that this doesn't make sense?

Reply:
I'm happy to add an extra example, in my mind I put the xct example at the start of the example and then I have the equivalent maui example later. I'll put them together so its clearer for the reader :)

Reply:
I've merged the XCT & Maui examples into the same section so its clear that A migrates to B